systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -540,7 +540,8 @@ void testScaleMirrorMaker2UpAndDownToZero() {
 
         LOGGER.info("Scaling subresource replicas to {}", scaleTo);
 
-        KubeResourceManager.get().kubeCmdClient().inNamespace(testStorage.getNamespaceName()).scaleByName(KafkaMirrorMaker2.RESOURCE_KIND, testStorage.getClusterName(), scaleTo);
+        // We use the KafkaMirrorMaker2.v1beta2.kafka.strimzi.io kind to use the v1beta2 API. It should be removed once the KafkaConnect CR used is a valid v1 resource.
```

Comment:
Should we create an issue for it (to replace it)? Or do you think that once the tests start failing, we will figure it out? :D